cmd/pdc/main.go

```diff
@@ -97,6 +97,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Allow overrides of _some_ empty flags with environment variables
+	clusterOverride, ok := os.LookupEnv("GCLOUD_PDC_CLUSTER")
```

Comment:
I suggest a different name to `Override`, because we're not really overriding here. Maybe `*EnvVar`? Same for the other new variables.

```suggestion
	clusterEnvVar, ok := os.LookupEnv("GCLOUD_PDC_CLUSTER")
```